CHANGELOG.md

```diff
@@ -0,0 +1,135 @@
+## To be released
```

Comment:
@sap1ens  Maybe we should update Mobitron to check if `develop` is ahead of `master`.  If it is and the CHANGELOG is empty, we complain loudly and abort? (in a future PR :) )